### DISCUSSION TITLE
Siembol presentation at Black Hat USA - August 2021

### DISCUSSION THREAD

**Discussion by c-rindi** [G-Research/siembol#119] [Events]
Siembol will have its inaugural public presentation at Black Hat 2021!

https://www.blackhat.com/us-21/arsenal/schedule/index.html#siembol-an-open-source-real-time-siem-tool-based-on-big-data-technologies-24038

Come hear Marian Novotny talk about this "Open-Source Real-Time SIEM Tool Based on Big Data Technologies".

Stay tuned for the exact date/time of the session, or come join us in Las Vegas. :)
Conference dates are July 31-Aug 5 at Mandalay Bay Hotel.

We will also be presenting at DEF CON a few days later.

Any conference video will be posted in this channel afterwards.

**Comment by c-rindi** [comment]
Session is on Thursday at 12pm noon PST, and will be a virtual presentation.

Details here: https://www.blackhat.com/us-21/arsenal/schedule/index.html#siembol-an-open-source-real-time-siem-tool-based-on-big-data-technologies-24038

**Comment by mariannovotny** [comment]
Siembol has been added to https://github.com/toolswatch/blackhat-arsenal-tools/blob/master/forensics/siembol.md

**Comment by c-rindi** [comment]
Siembol got mentioned in The Record, a "cybersecurity news publication", including a link to our first blog post. 
Read both at:
- https://therecord.media/security-tools-showcased-at-black-hat-usa-2021/
- https://www.gresearch.co.uk/article/introducing-siembol-an-open-source-real-time-security-information-and-event-management-tool/

**Comment by tabathad** [comment]
Watch the replay of the Seimbol BlackHat US 2021 presentation on the [GR-OSS YouTube channel](https://www.youtube.com/channel/UCI3GL6q9RUL3swr3HftS8Qg), [take a look](https://youtu.be/8S_EshQDTSg)!
